README.md

```diff
@@ -2,4 +2,99 @@
 Default implementation of Ruuvi Bluetooth library for Android
 
 # This link is very useful:
-https://github.com/ruuvi/ruuvi-sensor-protocols
\ No newline at end of file
+https://github.com/ruuvi/ruuvi-sensor-protocols
+
+# Usage example
+
+Add the JitPack repository to your projects build.gradle file
+```gradle
+allprojects {
+    repositories {
+        maven { url "https://www.jitpack.io" }
+        ...
+    }
+}
+```
+
+Add the library and its interface to your project by adding following as dependencies in app/build.gradle
+```gradle
+dependencies {
+    implementation 'com.github.ruuvi:com.ruuvi.bluetooth:2cedd919b2'
+    implementation 'com.github.ruuvi:com.ruuvi.bluetooth.default:48cf870335'
+    ...
+}
+```
+
+Add bluetooth and location permission in AndroidManifest.xml
+```xml
+<uses-permission android:name="android.permission.BLUETOOTH" />
+<uses-permission android:name="android.permission.BLUETOOTH_ADMIN" />
+<uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
+```
+
+You are now ready to use the library. You can get started by initializing the RuuviRangeNotifier, requesting location permission and start the scanning. You activity could look something like this:
+```kotlin
+import android.Manifest
+import android.content.pm.PackageManager
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import androidx.core.app.ActivityCompat
+import androidx.core.content.ContextCompat
+import com.ruuvi.station.bluetooth.FoundRuuviTag
+import com.ruuvi.station.bluetooth.IRuuviTagScanner
+import com.ruuvi.station.bluetooth.RuuviRangeNotifier
+
+class MainActivity : AppCompatActivity(), IRuuviTagScanner.OnTagFoundListener {
+    private lateinit var ruuviRangeNotifier: IRuuviTagScanner
```

Comment:
We should use BluetoothInteractor instead. Probably should make RuuviRangeNotifier private